### PR TITLE
Update Alibaba deletion instructions

### PR DESCRIPTION
**Pull request by abigailwillow**
#2171

### CODE REVIEW

**Comment by tupaschoal** [comment]
Given what you wrote on #2171, shouldn't it be changed to impossible?

**Comment by abigailwillow** [comment]
My apologies for the confusion, the previous instructions to deactivate would suggest that. But I found a way to permanently delete the account a few minutes later, which is why I opened this PR.

File: _data/sites.json [hunk 1]
```diff
@@ -961,9 +961,9 @@
 
     {
         "name": "Alibaba",
-        "url": "https://accounts.alibaba.com/user/organization/manage_person_profile.htm",
+        "url": "https://myprivacy.alibaba.com/home#/request/delete",
         "difficulty": "easy",
-        "notes": "To delete your account, go to \"My Account\" page, click on \"Member Profile\" and then, \"Deactivate Account\"",
+        "notes": "To delete your account, go to the Alibaba privacy center by going to myprivacy.alibaba.com/home#/request/delete, then click \"Delete Account\", enter \"delete\" in the text field, and click \"Delete Account\" again.",
```
Comment (tupaschoal) [inline _data/sites.json:966]:
```suggestion
        "notes": "To delete your account, go to the Alibaba privacy center linked above, then click \"Delete Account\", enter \"delete\" in the text field, and click \"Delete Account\" again.",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**